emc-metalnx-services/src/test/java/com/emc/metalnx/services/irods/CollectionServiceImplTest.java

```diff
@@ -140,5 +141,46 @@ public void testGetSubCollectionsAndDataObjectsUnderPathThatMatchSearchTextPagin
 		Assert.assertTrue("no recs returned", actual.size() > 1);
 
 	}
+	
+	@Test
+	public void testCreateCollectionWithParent() throws Exception {
+		
+		IRODSAccount irodsAccount = testingPropertiesHelper.buildIRODSAccountFromTestProperties(testingProperties);
+
+		IRODSServicesImpl irodsServices = new IRODSServicesImpl();
+		irodsServices.setIrodsAccount(irodsAccount);
+		irodsServices.irodsAccessObjectFactory = irodsFileSystem.getIRODSAccessObjectFactory();
+
+		CollectionServiceImpl collectionService = new CollectionServiceImpl();
+		collectionService.setIrodsServices(irodsServices);
+
+		// get new collection as DataGridCollectionAndDataObject
+		String newSubCollection = "subdir1/subdir2";
+		StringBuilder parentCollectionStringBuilder = new StringBuilder();
+		parentCollectionStringBuilder.append(irodsAccount.getHomeDirectory());
+		//parentCollectionStringBuilder.append("/subdir1");
```

Comment:
remove if unneeded

Reply:
Does the change need to be pushed to github?

L161 is still commented out.